frame/staking/src/pallet/mod.rs

```diff
@@ -345,6 +355,47 @@ pub mod pallet {
 	pub type Nominators<T: Config> =
 		CountedStorageMap<_, Twox64Concat, T::AccountId, Nominations<T>>;
 
+	/// A helper struct with some explicit functions for nominators that are existing in storage, but
+	/// cannot be decoded. See [`Nominators`] for more info.
+	pub struct NominatorsHelper<T>(sp_std::marker::PhantomData<T>);
```

Comment:
this was a huge hack and I would re-evaluate the need for its existence before bringing it in.

Reply:
Yeah, I was thinking about it too. I guess we still kinda need it for the migration to make sure it works. I have no other ideas on how to make sure that all the nominations are decodable at the time of migration. I could just introduce this struct intro migrations mod itself, but not sure it makes a difference.